### DISCUSSION TITLE
Tasmota sonoff ZBbridge, mqtt message ZbInfo, how?

### DISCUSSION THREAD

**Discussion by Hoeby** [arendst/Tasmota#14879] [Q&A] [answered]
I want to use the ZbInfo (or ZbStatus3), but not from the console, but with a mqtt message.
I tried but can't figger it out and can't find any information about it on www.

Does somebody know the working mqtt message for this?

Tried someting like this, without succes:
{ "topic": "cmnd/tasmota_96373D/ZbInfo","payload":{"Device":0x41CD}};

**Comment by rebel7580** [comment, marked as answer]
topic:  cmnd/tasmota_96373D/ZbInfo
payload: 0x41CD

If you have named this device, you can use the name instead of the short address (preferred way).

**Reply by sfromis** [reply]
In general, **all** the commands you can use in the console, can also be used via MQTT.
https://tasmota.github.io/docs/MQTT/#commands-over-mqtt
